Big-Spender/readme.md

```diff
@@ -68,39 +69,64 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends
+WHERE lower(description) LIKE '%fee%';
```

Comment:
what about transactions with the description "The Nurse's station Coffee Machine" or "1000x feeding tubes"